pyproject.toml

```diff
@@ -125,16 +119,14 @@ exclude_also = [
 
 [tool.pytest.ini_options]
 addopts = [
-    # "--import-mode=importlib",  # TODO: enable: https://github.com/scverse/anndata/issues/1451
+    "--import-mode=importlib",
```

Comment:
```suggestion
```

I'm getting quite a few errors locally, but removing this seems to solve that. Does removing this break anything?

Reply:
cannot reproduce, could you please try investigating why that is?